packaging/resource_suite_s3_cache.py

```diff
@@ -649,7 +649,7 @@ def setUp(self):
         if hasattr(self, 'static_bucket_name'):
             self.s3bucketname = self.static_bucket_name
         else:
-            distro_str = '{}-{}'.format(distro.id(), distro.version()).replace(' ', '').replace('.', '')
+            distro_str = ''.join(platform.linux_distribution()[:2]).replace(' ','').replace('.', '')
```

Comment:
Is `distro` not working here? I thought this was the preferred module these days...

Reply:
I don't think we moved to `distro` for 4-2-stable, so I think this is okay?

Willing to be persuaded otherwise.

Reply:
Oh, I see. This is only in the tests which were being skipped until now, and so it only now presents a problem because `distro` isn't being used/supported in this branch. As long as we don't do this in the cherry-pick to main, then this is fine.

Reply:
This failed which is why I made the change.